config/routes.d/staging.horse.json

```diff
@@ -4,7 +4,8 @@
       "^/docs/private-cloud/release-notes/": "developer.rackspace.com/docs-singlepage.html",
       "^/docs/private-cloud/rpc/eng-handbook/": "developer.rackspace.com/docs-singlepage.html",
       "^/docs/private-cloud/": "developer.rackspace.com/user-guide.html",
-      "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html"
+      "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html",
+      "^/docs/billing-apiguide/": "developer.rackspace.com/docs-singlepage.html"
```

Comment:
Most of the public API guides are using the user guide template (see [here](https://github.com/rackerlabs/nexus-control/blob/master/config/routes.d/developer.rackspace.com.json); the only ones using single page are one or two developer guides, a glossary, and release notes), so that's the one I would use here. It will be more indicative of the production view. To fix it, change `docs-singlepage.html` to `user-guide.html`.